src/PatchResolver.js

```diff
@@ -3,13 +3,14 @@ import { parseMultipartHttp } from './parseMultipartHttp';
 export function PatchResolver({ onResponse, boundary }) {
     this.boundary = boundary || '-';
     this.onResponse = onResponse;
-    this.processedChunks = 0;
     this.chunkBuffer = '';
+    this.isPreamble = true;
```

Comment:
can you explain what is and the importance of the `isPreamble`?

Reply:
Everything before the first boundary should be ignored. So this just tracks that we've passed the first boundary. Another way could have been to increment a number, and simply not yield if less than 1.